pkgs/by-name/li/lib60870/package.nix

```
@@ -0,0 +1,28 @@
+{ cmake
+, lib
+, stdenv
+, fetchzip
+}:
+stdenv.mkDerivation {
+  pname = "lib60870";
+  version = "2.3.1";
+
+  src = fetchzip {
+    url = "https://libiec61850.com/wp-content/uploads/2022/03/lib60870-2.3.1.tar.gz";
+    hash = "sha256-TQyWbfmFI+Dc5AMD0juogDrEIVnuYaPNQ7PT5ejJzjw=";
+  };
+
+  separateDebugInfo = true;
+
+  nativeBuildInputs = [ cmake ];
+
+  preConfigure = "cd lib60870-C";
+
+  meta = with lib; {
+    description = "Implementation of the IEC 60870-5-101/104 protocol";
+    homepage = "https://libiec61850.com/";
+    license = licenses.gpl3;
```

gpl3 is deprecated on favor of gpl3Only and gpl3Plus
```suggestion
    license = licenses.gpl3Only;
```

---

Thanks. Addressed in e1370a466ca9174fe8ba5f699321d4db48dfefb4